internal/clients/flag/flag.go

```diff
@@ -0,0 +1,19 @@
+package flag
+
+import "github.com/spf13/pflag"
+
+// Values contains all values parsed from command line flags
+type Values struct {
+	Debug bool
+}
+
+// Parse parses flags from command line
+func Parse() *Values {
+	values := Values{}
+
+	pflag.BoolVarP(&values.Debug, "debug", "d", false, "Enable debug logs")
```

Comment:
I assume this func is doing something along the lines of looking for `debug` or `d` CLI arguments and adding whether it was found or not (plus any value following it) to the values struct? If so, I think it probably makes sense to const all these strings at the top of the file for now so that it's super clear what each of them does and so that they're easy to change as we extend the flag functionality.